lib/ansible/plugins/callback/profile_tasks.py

```diff
@@ -12,20 +12,20 @@
 DOCUMENTATION = '''
     callback: profile_tasks
     type: aggregate
-    short_description: adds time information to tasks
+    short_description: adds time and summary information to tasks
     version_added: "2.0"
     description:
       - Ansible callback plugin for timing individual tasks and overall execution time.
-      - "Mashup of 2 excellent original works: https://github.com/jlafon/ansible-profile,
-         https://github.com/junaid18183/ansible_home/blob/master/ansible_plugins/callback_plugins/timestamp.py.old"
-      - "Format: C(<task start timestamp> (<length of previous task>) <current elapsed playbook execution time>)"
+      - 'Mashup of 2 excellent original works: U(https://github.com/jlafon/ansible-profile),
+        U(https://github.com/junaid18183/ansible_home/blob/master/ansible_plugins/callback_plugins/timestamp.py.old)'
+      - 'Format: C("<task start timestamp> (<length of previous task>) <current elapsed playbook execution time>")'
       - It also lists the top/bottom time consuming tasks in the summary (configurable)
       - Before 2.4 only the environment variables were available for configuration.
     requirements:
       - whitelisting in configuration
     options:
-      output_limit:
-        description: Number of tasks to display in the summary
+      task_output_limit:
```

Comment:
why are you renaming the option? no alias, no deprecation?

Reply:
@bcoca I don't believe this actually renames the option?  This is just the name that you look it up with in the code (i.e. ```self.get_option('output_limit')```) but to actually *set* it you put in ansible.cfg

```
[callback_profile_tasks]
task_output_limit = all
```

I can not see any other way to set it by the reference ```output_limit``` but ... please point it out to me :)  When you read the documentation page (https://docs.ansible.com/ansible/2.5/plugins/callback/profile_tasks.html) I think it makes it quite confusing having big and bold ```output_limit``` but then you have to actually set ```task_output_limit```

Reply:
indentation fixed

Reply:
changes the documented option, what users can search for .. also anyone using derived plugin will be affected

Reply:
@bcoca Sorry I'm just a little unsure what you want to see here.  I'm not sure what you mean by a derived plugin either.

I will add an alias for it; but I am equally happy to just drop this all together and leave it as is (the only reason I proposed this was because I didn't read this closely enough, and at first thought the ini-setting was ```output_limit```).  

AFAICT, there is no way that any code could have had ```output_limit = all``` in any way, but again I may be misunderstanding how you can actually set callback options (from my understanding, these can only either from ansible.cfg -- where you must use the ini-key ```task_output_limit``, or the environment variable).